tests/unit/test_launchers.py

```diff
@@ -178,8 +178,126 @@ def test_launcher_deleted_whith_statement(self):
         self.assertFalse(os.path.isdir(tempdir))
 
 
-@pytest.mark.skipif(sys.platform == "darwin", reason="fails on macosx")
```

Comment:
FYI [this change](https://github.com/mozilla/mozregression/pull/1080/files#diff-7246e1ea17f70a16641a4c92fd5483979a1bb1297111644f6e68886b19201ef5R239) was made as part of fixing the failures of this test (`test_firefox_install`).